2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,14 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  let sutibleAricleTiteles = [];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length <= 65) {
```

Comment:
This is great, you could do this with some other functions but this works and isn't a performance drain.